### PR TITLE
add support for remote vlan breakout mechanism

### PR DESCRIPTION
**Pull request by pperiyasamy**
Depends on https://github.com/networkservicemesh/sdk-ovs/pull/90

Signed-off-by: Periyasamy Palanisamy <periyasamy.palanisamy@est.tech>

### CODE REVIEW

File: internal/devicecfg/devicecfg.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/networkservicemesh/sdk-sriov/pkg/tools/yamlhelper"
+	"github.com/networkservicemesh/sdk/pkg/tools/log/logruslogger"
+)
+
+// Config contains list of available service domains
```
Comment (ljkiraly) [inline internal/devicecfg/devicecfg.go:31]: Please reword this comment. The config stores a list of devices. It is faulty in vpp forwarder also.

File: internal/devicecfg/devicecfg.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+type Config struct {
+	Interfaces []*Resource `yaml:"interfaces"`
```
Comment (ljkiraly) [inline internal/devicecfg/devicecfg.go:33]: Do you need interfaces for ovs also?

Comment (pperiyasamy) [reply internal/devicecfg/devicecfg.go]: @ljkiraly yes, the l2 connection point is either existing physical interface or existing ovs bridge, when interface name is provided, then ovs forwarder creates a normal ovs bridge (the latest 'config' file takes bridge name as input parameter under 'interface') and adds that interface as ovs port into the bridge.

File: internal/devicecfg/devicecfg.go (hunk 1 above)
Comment (pperiyasamy) [reply internal/devicecfg/devicecfg.go]: changed it, is that fine ?

**Pull request merged**